packages/edge/src/edge-headers.ts

```diff
@@ -68,6 +71,16 @@ function getHeader(request: Request, key: string): string | undefined {
   return request.headers.get(key) ?? undefined;
 }
 
+function getFlag(countryCode: string | undefined): string | undefined {
+  if (!countryCode) return undefined;
```

Comment:
<!-- Questions comments seek clarification.  -->
**question (non-blocking):** ‏Should there be a validation that the string is not above a certain length or contains certain allowed characters?

Reply:
Good point. This function will work providing the return value of `getHeader(request, COUNTRY_HEADER_NAME)` is a two-digit country code. There's nothing currently in place to check the return values of `getHeader`, I assume because it'll depend on what's passed in?

I'll add a check and test to ensure `countryCode` is only two-digits.